src/file_organizer/updater/trust.py

```
@@ -0,0 +1,109 @@
+        expected_version: Expected normalised version (e.g. 2.1.0).
+
+    Returns:
+        The verified manifest data as dict, or None if validation fails.
+    """
+    if not verify_manifest_signature(manifest_content, signature_content):
+        logger.error("Manifest signature verification failed.")
+        return None
+
+    try:
+        manifest_data = json.loads(manifest_content)
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)